src/main/java/org/broadinstitute/hellbender/utils/nio/PathLineIterator.java

```diff
@@ -62,4 +76,8 @@ public void forEach(Consumer<? super String> action) {
     public Spliterator<String> spliterator() {
         return lines.spliterator();
     }
+
+    public Stream<String> stream() {
```

Comment:
This appears to be unused. I would remove it since it can have the side effect of consuming the stream out from underneath the iterator.

Reply:
OK.